raw_data/preprocessed/africa/gambia_english.txt

```diff
@@ -0,0 +1,221 @@
+
+Offences relating to records
+(54) [1] A person commits an offence if he: [a] fails to keep records of the activities, products, by-products, and wastes required to be kept by this Act; or [b] fraudulently alters any record required by this Act. [2] A person who commits an offence under subsection [1] is liable on conviction: [a] in the case of a natural person, to a fine of not more than ten thousand dalasis or to a term of imprisonment of not more than two years; and [b] in the case of a body corporate, to a fine of not more than thirty thousand dalasis.
+
+Offences relating to environmental standards and guidelines
+(55) [1] A person commits an offence if he: [a] violates any environmental standard established in accordance with section 28; [b] violates any guideline made under this Act; or [c] uses natural resources in a wasteful manner contrary to measures and regulations prescribed under Part VII. [2] A person who commits an offence under subsection [1] is liable on conviction: [a] in the case of a natural person, to a fine of not more than twenty thousand dalasis or to a term of imprisonment of not more than five years; and [b] in the case of a body corporate, to a fine of not more than fifty thousand dalasis. 
+
+Offences relating to hazardous and dangerous materials, etc
+(56) A person commits an offence if he [a] fails to manage hazardous and dangerous materials, processes and wastes in accordance with the provisions of section 36; [b] knowingly mislabels any wastes, chemical or dangerous process or other materials; [c] withholds information about the management of wastes, chemicals or dangerous processes and substances; or [d] aids or abets the illegal traffic in wastes, chemicals or dangerous processes and substances. [2] A person who commits an offence under subsection [1] is liable on conviction: [a] in the case of a natural person, to a fine of not more than one hundred thousand dalasis or to a term of imprisonment of not more than six years; [b] in the case of a body incorporate, to a fine of not more than five hundred thousand dalasis.
+
+Offences relating to pollution
+(57) [1] A person commits an offence if he [a] discharges any dangerous materials, substance, oil or mixture containing oil into any waters or any other segment of the environment contrary to section 38; or [b] pollutes the environment contrary to section 39. [2] A person who commits an offence under subsection [1] is liable on conviction: [a] in the case of a natural person, to a fine of not more than two hundred thousand dalasis or to term of imprisonment of not more than ten years. [b] in the case of a body corporate, to a fine of not more than one million dalasis. 
```

Comment:
Please replace `he [a]` with `he: [a]`. Also, please replace `to term` with `to a term`. Furthermore, please replace `years. [b]` with `years; [b]`.